OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -46,5 +46,19 @@ public void ReturnToBase(Actor airfield = null)
 			else
 				Self.QueueActivity(new HeliReturnToBase(Self));
 		}
+
+		[ScriptActorPropertyActivity]
+		[Desc("Queues a landing activity on the specififed actor.")]
+		public void Land(Actor landOn)
+		{
+			Self.QueueActivity(new Land(Self, Target.FromActor(landOn)));
```

Comment:
Isn't that already done with `target.IsValidFor(self)` at https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Activities/Air/Land.cs#L31